lib/file-operations.js

```diff
@@ -66,6 +66,48 @@ function getTimesDiff(fsStat, vinylStat) {
   return timesDiff;
 }
 
+function getOwnerDiff(fsStat, vinylStat) {
+  if (/^win/.test(process.platform)) {
```

Comment:
there should be no reason to do this.  In `updateMetadata`, we exit early on windows (it doesn't have `process.getuid()`)